src/components/ec-radio-btn/types.ts

```diff
@@ -0,0 +1,19 @@
+export enum RadioButtonEvent {
+    UPDATE_MODEL_VALUE = 'update:modelValue',
+  }
+
+export interface RadioButtonEvents {
+    [RadioButtonEvent.UPDATE_MODEL_VALUE]: RadioButtonProps['modelValue']
+  }
+export interface RadioButtonProps {
+    modelValue?: string,
+    label?: string,
+    description?: string,
+    isDisabled?: boolean,
+    isTextInline?: boolean
+    value: string,
+    name?: string,
```

Comment:
```suggestion
    name: string,
```
It should be a required prop

Reply:
Please don't remember to make the same changes in the .vue file.

Reply:
it's not necessary to have it as a required prop